tests/lib/rules/no-global-jquery.js

```diff
@@ -457,5 +457,21 @@ ruleTester.run('no-global-jquery', rule, {
         },
       ],
     },
+    {
+      code: `
+        export default Ember.Component({
```

Comment:
Remember to include the Ember import statement for this. Alternatively, you could just simplify the entire test case to `jQuery.extend()` since the rule does not actually care if the Ember component is involved and the Ember component just adds unnecessary noise to the test case.